src/query/functions/temporal/base.go

```diff
@@ -341,7 +341,7 @@ func getIndices(
 			l = i
 		}
 
-		if ts.Before(rBound) {
+		if !ts.After(rBound) {
```

Comment:
Hm this switches this behavior to be inclusive of the upper bound yeah?

Can we update the `getIndices` method comments to include that both lbound and rbound are inclusive? 

Will this have correct effect in terms of not double counting datapoints if you split up some range, say `[a,c]`, as `[a,b]` and `[b,c]` then do a sum of the final result together?

Reply:
Yeah Prom ends up double-counting them if they're on the step boundary, so figured we should match to that for now (could put in a check later for not double-dipping if we add in a different QL)

Reply:
Ah wow, ok no problems. Sounds good, so weird w.r.t Prom =/